projects/2021_05_09_user_operated_internet_fund.md

```diff
@@ -508,6 +508,187 @@ Metrics regarding applications to the October call
 * Number of applications: **33**
 * Total budget requested: **1,124,191**€
 
+## M3
+
+> This milestone will be delivered no later than December 6th 2021 and will include the selected grantees from the October call as well as metrics regarding applications to the December call and information about ongoing projects, the necessary PKT to fund the new projects will be converted to EUR as the projects are accepted.
+
+> Another call ("The February call") will open on December 1st with deadline February 1st.
+
+> After a report is delivered on this milestone and if it is approved by the NS, the NS will send the amount of PKT needed to raise the balance of the Trust's treasury to the Buffer Amount.
+
+
+We are submitting M3 on March 8th 2022. This includes the results from the 2021-10U (October 1st) and 2021-12U (December 1st), which were merged due to financial delays. The two rounds combined had 43 proposals.
+
+Six new projects were granted, bringing the total to 10 regular projects originating from 7 countries: Argentina (1 proposal, 9%), Germany (4 proposals, 36%), South Korea (1 proposal, 9%), Mexico (1 proposal, 9%), Slovenia (1 proposal, 9%), United States (2 proposals, 18%), South Africa (1 proposal, 9%). 
+
+The total amount requested so far was € 3 031 600, with applicants from 29 countries that together submitted 87 projects. 
+
+
+### 1. The balance of payments
+
+The balance of payments to the User Operated Internet Fund
+
+- 20mn PKT was donated by the Network Steward so far.
+
+- 20mn PKT was converted into € 377 240. See #4.
+
+The current balance is 0 PKT.
+
+### 2. Converted from PKT
+
+The amount of money which was converted from PKT during the reported period
+
+    10mn PKT -> 159740,16 (180.000 dollar) 
```

Comment:
Can this be changed to match the formatting used throughout the  rest of the document? eg. `$180,000`